docs/docs/community/index.mdx

```diff
@@ -0,0 +1,97 @@
+
+All contributions are welcome, including issues, contributing code, new docs as well as updates and tweaks, blog posts,
+helping out people, organizing community events, working on accessibility and design items, and more.
+Continue reading to learn what the community can do for you and what you can do for the community.
+By contributing to open source projects, you can connect with people, learn new skills, become a subject-matter expert,
+and apply all learnings to your projects.
+
+## Resources
+
+### Code of conduct
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make
+participation in our project and our community a harassment-free experience for everyone, regardless of age, body size,
+disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+Read the Nebari [Code of Conduct][nebari-coc].
+
+### Getting support
+
+- [GitHub Discussions][nebari-discussions] serve as our user forum. You can use them to raise discussions about a subject, such as:
+  "What is the recommended way to do _X_ with Nebari?"
+
+- [The Nebari issue tracker][nebari-issues] is the preferred channel for bug reports, documentation requests, and submitting pull requests.
+
+## How to contribute
+
+We want to make the contributing experience within Nebari fun, enjoyable, and educational for anyone and everyone.
+Contributions go far beyond pull requests and commits if you're interested in participating in the Nebari community.
+
+There are many ways in which you can help Nebari (they're all important, so we list them in alphabetical order):
+
+1. Code maintenance and development
+2. Community coordination and contributor experience
+3. DevOps
+4. Fundraising
+5. Marketing
+6. Project management
+7. Content translation and internationalization
+8. Website design and development
+9. Writing technical documentation
+10. UI/UX
+11. Opening issues and reviewing Pull Requests
+
+### Help fellow users
+
+Answering questions for fellow Nebari users on [GitHub Discussions][nebari-discussions] are valuable contributions.
+You can also help users on [issues][nebari-issues] and [Pull Requests][nebari-prs] to contribute or fix their issues.
+
+### Contribute documentation
+
+You can help make the docs awesome. We strive to create inclusive, comprehensive, and enjoyable documentation for our users and everyone can help!
+Found a typo or an incorrect code example? Put up a PR! Found a better way to explain a concept?
+Put up a PR! Contributions to our documentation are valuable and greatly appreciated.
+
+<!-- TODO: add link to doc contributions section-->
+
+Visit the [Documentation contributions section](doc-contributions.md) to learn more about our style guide, the documentation structure, and how to write docs.
```

Comment:
We have a CI failure because this doc doesn't exist yet

```suggestion
Visit the [Documentation contributions section]() to learn more about our style guide, the documentation structure, and how to write docs.
```